Source/Core/Settings.cs

```diff
@@ -56,6 +56,10 @@ public class Settings : SettingsBase<Settings> {
 
         public bool EnableElasticsearchTracing { get; private set; }
 
+		public string LdapConnectionString { get; private set; }
+
+		public bool EnableActiveDirectoryAuth { get; internal set; }
```

Comment:
I think this should be like some of our other enabled properties and just return true of the ldap connection string is set.